tardis/adapters/sites/slurm.py

```diff
@@ -174,7 +174,9 @@ def sbatch_cmdline_options(self, drone_uuid, machine_meta_data_translation_mappi
         walltime = self.machine_type_configuration.Walltime
 
         drone_environment = ",".join(
-            f"TardisDrone{key}={value}"
+            f"TardisDrone{key}={int(value)}"
+            if isinstance(value, float)
+            else f"TardisDrone{key}={value}"
```

Comment:
Can `value` be any non-numerical data type? If no, then this distinction between `float` and other numerical data-types might not be needed.

Reply:
It might be easier to always convert it to ``int``, no matter the type.

Reply:
Since it can be a string, that would fail.

Reply:
To be more precise, a string not convertible to int.

Reply:
I have added a convert_to function handling the conversion internally. So everything convertible to `int` is converted, strings that are not convertible are still handled as strings. For example the unique name of the drone.